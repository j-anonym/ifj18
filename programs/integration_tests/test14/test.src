a = 0.0 

b = 5.0
x = a / b
print x ,"\n"

x=1
while(x<=5) do
    x = x+1
    if(x == 4) then
    else
    end
print x
end

x=2
while(x<5) do
print x
x = x+1
if(x==5) then
else
end
end
