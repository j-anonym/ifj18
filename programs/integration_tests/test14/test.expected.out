0x0p+0
23456234